Reference/Config/404handlers/index.md

```diff
@@ -1,7 +1,20 @@
-#This section is waiting for content
+#404handlers.config
+
+Configuration file for legacy *NotFoundHandlers* - to register custom code within the Umbraco incoming request pipeline, including the legacy way to handle a custom 404 error.
+
+The Default Handlers are listed in this configuration file for 'backwards compatibility':
+
+    <NotFoundHandlers>
+      <notFound assembly="umbraco" type="SearchForAlias" />
+      <notFound assembly="umbraco" type="SearchForTemplate"/>
+      <notFound assembly="umbraco" type="SearchForProfile"/>
+      <notFound assembly="umbraco" type="handle404"/>
+    </NotFoundHandlers>
+
+The concept of *NotFoundHandlers* has been replaced by *IContentFinders* - [Implementing an IContentFinder](../../routing/request-pipeline/IContentFinder)
+
+Custom 404's are now handled by creating a custom IContentFinder registered as a *ContentLastChanceFinder* - [Using an IContentFinder for Custom 404s](../../routing/request-pipeline/IContentFinder#notfoundhandlers)
```

Comment:
You are suggesting there are multiple 404's possible with ContentLastChange, while this is the only and last one.  Every content finder can be a custom 404.